Add List of projects and their Leaders

rexim opened the pull request:
Close #6

rexim committed:
Add List of projects and their Leaders (#6)

rexim committed:
Enable user links (#6)

ForNeVeR commented on Projects.md:
```diff
@@ -0,0 +1,40 @@
+- Project Leader: [@06needhamt]
+- URL: https://github.com/morganey-lang/morganey-vscode-extension
+
+# organization #
```
I suggest to move that on top.

rexim replied:
Done!

rexim committed:
Move organization project on top (#6)

rexim committed:
Link refer to Projects.md from README.md (#6)

rexim committed:
Try to make link to Projects.md clickable (#6)

Used
http://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file
as reference

rexim committed:
Fix typo in README (#6)

rexim commented:
@ForNeVeR thanks for the review!
